addons/crm/models/lead_scoring_frequency.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from odoo import fields, models
+
+
+class LeadScoringFrequency(models.Model):
+    _name = 'lead.scoring.frequency'
+    _description = 'Lead Scoring Frequency'
+
+    variable = fields.Char('Variable')
+    value = fields.Char('Value')
+    won_count = fields.Float('Won Count')
```

Comment:
Weird to have a float but I understand it's for the + 0.1 thing...
Maybe add a comment for that?